MOTSChallenge
track_rcnn
Traceback (most recent call last):
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/eval.py", line 86, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/eval.py", line 191, in eval_sequence
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/datasets/_base_dataset.py", line 92, in get_raw_seq_data
    raw_gt_data = self._load_raw_file(tracker, seq, is_gt=True)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/datasets/mots_challenge.py", line 182, in _load_raw_file
    from pycocotools import mask as mask_utils
ModuleNotFoundError: No module named 'pycocotools'





MOTSChallenge
track_rcnn
Traceback (most recent call last):
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/eval.py", line 86, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/eval.py", line 191, in eval_sequence
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/datasets/_base_dataset.py", line 92, in get_raw_seq_data
    raw_gt_data = self._load_raw_file(tracker, seq, is_gt=True)
  File "/home/guest/dev_repo/SoccerTrack/sandbox/mot_eval_scripts/../trackeval/datasets/mots_challenge.py", line 182, in _load_raw_file
    from pycocotools import mask as mask_utils
ModuleNotFoundError: No module named 'pycocotools'





MOTSChallenge
track_rcnn
Traceback (most recent call last):
  File "/tmp/ipykernel_5154/3235724239.py", line 73, in <module>
    curr_res = {seq_key: seq_value[c_cls][metric_name] for seq_key, seq_value in res.items() if
  File "/tmp/ipykernel_5154/3235724239.py", line 73, in <dictcomp>
    curr_res = {seq_key: seq_value[c_cls][metric_name] for seq_key, seq_value in res.items() if
KeyError: 'HOTA'





MOTSChallenge
track_rcnn
Traceback (most recent call last):
  File "/tmp/ipykernel_5154/2836190967.py", line 73, in <module>
    curr_res = {seq_key: seq_value[c_cls][metric_name] for seq_key, seq_value in res.items() if
  File "/tmp/ipykernel_5154/2836190967.py", line 73, in <dictcomp>
    curr_res = {seq_key: seq_value[c_cls][metric_name] for seq_key, seq_value in res.items() if
KeyError: 'CLEAR'





